app/api/sessions.py

```diff
@@ -105,16 +105,15 @@ def query(self, view_kwargs):
         """
         query_ = self.session.query(Session)
         if view_kwargs.get('track_id') is not None:
-            track = safe_query(self, Track, 'id', view_kwargs['track_id'], 'track_id')
+            track = safe_query(Track, 'id', view_kwargs['track_id'], 'track_id')
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [No value for argument 'parameter_name' in function call](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=5526478)